src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcEndpointConfig.cs

```diff
@@ -57,6 +58,12 @@ public IpcEndpointConfig(string address, TransportType transportType, PortType p
                         throw new PlatformNotSupportedException($"{UnixDomainSocketSchema} is not supported on Windows, use {NamedPipeSchema}.");
                     break;
                 }
+                case TransportType.TcpClient:
+                {
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I don't think we need to block this here. I am not aware of any technical reason Windows couldn't support TCP and in terms of mitigating security risks we can keep the DiagnosticsClient constructor internal so that people don't call it. In the future if/when we are ready to expose it we can decide what is the proper scope we want to support.

Reply:
Done, now it is just a `break;` to not throw for TCP/IP